core/modules/update/update.fetch.inc

```diff
@@ -54,12 +54,13 @@ function update_fetch_data_batch(&$context) {
   // Grab another item from the fetch queue.
   for ($i = 0; $i < 5; $i++) {
     if ($item = $queue->claimItem()) {
+      $item_name = $item->data['project_type'] == 'core' ? t('Backdrop CMS core') : $item->data['info']['name'];
```

Comment:
```suggestion
      $item_name = ($item->data['project_type'] == 'core') ? t('Backdrop CMS') : $item->data['info']['name'];
```

To make this easier to read, can we add brackets around the condition part? Also, considering `theme_update_report()`, let's use the same term for core: Backdrop CMS.